noetic/distribution.yaml

```diff
@@ -9346,7 +9346,7 @@ repositories:
     source:
       type: git
       url: https://github.com/robotraconteur/robotraconteur.git
-      version: ros-noetic
+      version: ros
```

Comment:
Same question about this change:

https://github.com/robotraconteur-packaging/robotraconteur-ros-release/blob/ce2b79da28408d28083121be912bfcce246589a1/tracks.yaml#L17

Reply:
See above